CHANGELOG.md

```diff
@@ -8,10 +8,15 @@
 
 ### Bug fixes
 
+* When loading Qiskit circuits to PennyLane templates using `load` in `converter.py`, parameters with `requires_grad=False` are bound to the circuit.  The old version bound objects that were not PennyLane `Variable`'s, but that object class is now deprecated.
```

Comment:
Maybe this description could be shortened and rephrased a bit? It's worth noting that not only those parameters with `requires_grad=False` can be bound to the circuit. The `_extract_variable_refs` function actually tries to gather the values for each trainable parameter (`requires_grad=True`) such that the value of those parameters can be bound to the circuit.